chia/full_node/mempool.py

```diff
@@ -15,6 +16,7 @@ def __init__(self, max_size_in_cost: int):
         self.removals: Dict[bytes32, MempoolItem] = {}
         self.max_size_in_cost: int = max_size_in_cost
         self.total_mempool_cost: int = 0
+        self.fee_estimator = FeeEstimatorDemo()
```

Comment:
"Demo" makes it sound like this is temporary, and not production. But, yet, it's hooked up to production code

Reply:
Yes, what I did was split this PR into 3 parts so it would be easier to review.

But I needed some glue code at the points where the other PRs connect.

Maybe I should also push this as one giant PR as well, so we'll have the exact code we will commit.